### ISSUE TITLE
Class support for icons / awesomeIcons

### ISSUE THREAD

**Issue by MSzturc** [TheCoder4eu/BootsFaces-OSP#70]
It seams that class property is not evaluated on icon / awesomeIcon tag.

Example:
```
<b:icon name="envelope" class="hidden-md"/>
```
produces the following html code:

```
<span id="login:j_idt69_input-group-addon" class="input-group-addon">
     <i id="login:j_idt69_icon" class="glyphicon glyphicon-envelope"></i>
</span>
```

i would expect:

```
<span .... class="input-group-addon hidden-md">
```

**Comment by TheCoder4eu** [comment]
Hi,
```<b:icon>```
still does not support the styleClass standard JSF attribute.
We can add it in the next release.

**Comment by stephanrauh** [comment]
Currently Yeray is working on the ticket.

**Comment by stephanrauh** [comment]
@yersan solved this ticket by providing pull request #118. Thank, Yeray!
